syntaxes/d2.tmLanguage.json

```diff
@@ -87,7 +87,7 @@
           "end": "\\1\\|",
           "patterns": [
             {
-              "include": "text.html.markdown.d2"
+              "include": "text.html.markdown"
```

Comment:
this is what I'm getting testing with the files from #53

### before 
![Screen Shot 2023-03-01 at 2 56 09 PM](https://user-images.githubusercontent.com/85081687/222286013-16cb8dbe-82c6-403f-bcd1-b78ecff1bfce.png)

### after
![Screen Shot 2023-03-01 at 2 55 39 PM](https://user-images.githubusercontent.com/85081687/222286034-b1774a8d-7d59-4a8a-9242-d6bc15bf24ed.png)

Reply:
I backed out my changes and have decided to leave well enough alone.  I can't confirm, but there might be a nested syntax limit in vscode with injected grammars.